docker/full-tls-mutinynet/docker-compose.yaml

```diff
@@ -32,7 +32,7 @@ services:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
 
   fedimintd:
-    image: fedimint/fedimintd:master
+    image: fedimint/fedimintd:v0.1.0-rc3
```

Comment:
This doesn't look sustainable. I should have thought of that when ACKing #3044. I still think an `v0.1-latest` tag would be better. Sure, as @dpc pointed out the image will get cached, but if you really want the latest version you should probably just manually pull it again.

Reply:
I think any upgrade to a federation should be done carefully and manually. So here we should specify a very specific tag/hash.